### PR TITLE
Explain how to extract the cylc wrapper in installation.rst

### PR DESCRIPTION
**Pull request by wxtim**
Companion of https://github.com/cylc/cylc-flow/pull/4536

### CODE REVIEW

File: src/installation.rst [hunk 1]
```diff
@@ -179,13 +179,15 @@ available too.
...
 
-.. important::
+.. TODO - make this easier once extract-resources is better.
 
-   Cylc comes with a wrapper that you can use with minimal adaptation.
-   This should be installed somewhere in the system search ``$PATH`` such
-   as ``/usr/local/bin``.
+.. code-block:: bash
+
+   cylc resources etc/cylc /path/to/cylc  # should be in $PATH
```
Comment (datamel) [inline src/installation.rst:189]: Needs updating?
```suggestion
   cylc get-resources etc/cylc /path/to/cylc  # should be in $PATH
```

File: src/user-guide/writing-workflows/configuration.rst [hunk 2]
```diff
@@ -130,24 +130,24 @@ Atom
 Emacs
    The syntax file can be obtained from the Cylc library by
    running the following command
-   ``cylc extract-resources . etc/syntax/cylc-mode.el``
+   ``cylc resources . etc/syntax/cylc-mode.el``
```
Comment (datamel) [inline src/user-guide/writing-workflows/configuration.rst:133]: These need a `get-` on them?

File: src/installation.rst [hunk 3]
```diff
@@ -170,13 +170,15 @@ available too.
 We recommend using a wrapper script named ``cylc`` to activate the correct
 environment before calling the environment's  ``cylc`` command.
 
-.. TODO - update this once the wrapper has been added to cylc-flow package data.
+Cylc comes with a wrapper which can be extracted to a directory in your $PATH
+using
```
Comment (hjoliver) [inline src/installation.rst:174]: It doesn't have to be a directory in your `$PATH`, does it?  Just give any target path on the command line.

File: src/user-guide/writing-workflows/configuration.rst (hunk 2 above)
Comment (wxtim) [reply src/user-guide/writing-workflows/configuration.rst]: Fixed this

File: src/user-guide/writing-workflows/configuration.rst [hunk 4]
```diff
@@ -130,24 +130,24 @@ Atom
 Emacs
    The syntax file can be obtained from the Cylc library by
    running the following command
-   ``cylc extract-resources . etc/syntax/cylc-mode.el``
+   ``cylc get-resources . etc/syntax/cylc-mode.el``
```
Comment (datamel) [inline src/user-guide/writing-workflows/configuration.rst:133]:
```suggestion
   ``cylc get-resources etc/syntax/cylc-mode.el .``
```

File: src/user-guide/writing-workflows/configuration.rst [hunk 5]
```diff
@@ -130,24 +130,24 @@ Atom
...
    installation instructions are at the top of the file.
 Gedit
    The syntax file can be obtained from the Cylc library by
    running the following command
-   ``cylc extract-resources . etc/syntax/cylc.lang``
+   ``cylc get-resources . etc/syntax/cylc.lang``
```
Comment (datamel) [inline src/user-guide/writing-workflows/configuration.rst:138]:
```suggestion
   ``cylc get-resources etc/syntax/cylc.lang .``
```

File: src/user-guide/writing-workflows/configuration.rst [hunk 6]
```diff
@@ -130,24 +130,24 @@ Atom
...
    installation instructions are at the top of the file.
 Kate
    The syntax file can be obtained from the Cylc library by
    running the following command
-   ``cylc extract-resources . etc/syntax/cylc.xml``
+   ``cylc get-resources . etc/syntax/cylc.xml``
```
Comment (datamel) [inline src/user-guide/writing-workflows/configuration.rst:143]:
```suggestion
   ``cylc get-resources etc/syntax/cylc.xml .``
```

File: src/user-guide/writing-workflows/configuration.rst [hunk 7]
```diff
@@ -130,24 +130,24 @@ Atom
...
    installation instructions are at the top of the file.
 PyCharm
    Install the `Cylc.tmbundle`_.
 Vim
    The syntax file can be obtained from the Cylc library by
    running the following command
-   ``cylc extract-resources . etc/syntax/cylc.vim``
+   ``cylc get-resources . etc/syntax/cylc.vim``
```
Comment (datamel) [inline src/user-guide/writing-workflows/configuration.rst:150]:
```suggestion
   ``cylc get-resources etc/syntax/cylc.vim .``
```

**Review by datamel — COMMENTED** [review]
The order needs changing as the dir and resources has switched. Otherwise grand, thanks @wxtim.

**Comment by wxtim** [comment]
Good spot @datamel

**Review by datamel — APPROVED** [review]
Thanks @wxtim.

File: src/installation.rst [hunk 8]
```diff
@@ -170,13 +170,15 @@ available too.
...
 
-.. important::
+.. TODO - make this easier once extract-resources is better.
```
Comment (MetRonnie) [inline src/installation.rst:176]: Minor but could avoiud confusion in future
```suggestion
.. TODO - make this easier once get-resources is better.
```

File: src/installation.rst [hunk 9]
```diff
@@ -170,13 +170,15 @@ available too.
 We recommend using a wrapper script named ``cylc`` to activate the correct
 environment before calling the environment's  ``cylc`` command.
 
-.. TODO - update this once the wrapper has been added to cylc-flow package data.
+Cylc comes with a wrapper which can be extracted to a directory in your $PATH
```
Comment (MetRonnie) [inline src/installation.rst:173]:
```suggestion
Cylc comes with a wrapper which can be extracted to a directory in your ``$PATH``
```

**Review by MetRonnie — APPROVED** [review]
Feel free to merge with/without minor points addressed,

**Comment by hjoliver** [comment]
Merging, with @MetRonnie 's minor corrections applied.  Thanks @wxtim

**Pull request merged**